nixos/modules/programs/ryzen-monitor-ng.nix

```diff
@@ -0,0 +1,39 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.ryzen_monitor_ng;
+
+in
+{
+
+  options = {
+
+    programs.ryzen_monitor_ng = {
+      enable =  mkEnableOption (lib.mdDoc ''
+        A userspace application for setting and getting Ryzen SMU (System Management Unit) parameters via the ryzen_smu kernel driver.
+
+        Monitor power information of Ryzen processors via the PM table of the SMU.
+
+        SMU Set and Get for many parameters and CO counts.
+
+        https://github.com/mann1x/ryzen_monitor_ng
+
+        WARNING: Damage cause by use of your AMD processor outside of official AMD specifications or outside of factory settings are not covered under any AMD product warranty and may not be covered by your board or system manufacturer's warranty.
+      '');
+    };
+
+  };
+
+  config = mkIf cfg.enable (mkMerge [
```

Comment:
why `mkMerge` here?

Reply:
Copy paste from another package.

That's a terrible reason : P

I'm going to read up on it now.

Reply:
https://nixos.org/manual/nixos/stable/#sec-option-definitions-merging

Not needed, removing.

Thanks!